src/category/service.ts

```diff
@@ -0,0 +1,92 @@
+import { User } from 'src/auth'
+import { Category } from './category'
+import { CategoryLevel } from './level'
+import { CategoryItem } from './item'
+import { ImportSummary } from './importSummary'
+
+export interface CategoryImportJob {
+  // Todo
+}
+
```

Comment:
let's split into 3. CategoryService, CategoryLevelService, CategoryItemService.
so we avoid prefix like level, item to method names.